plugins/Overlay/API.php

```diff
@@ -40,19 +41,13 @@ public function getTranslations($idSite)
     /**
      * Get excluded query parameters for a site.
      * This information is used for client side url normalization.
+     *
+     * @deprecated use SitesManager.getExcludedQueryParameters instead
+     * @todo Remove in Matomo 6
      */
     public function getExcludedQueryParameters($idSite)
     {
-        $sitesManager = APISitesManager::getInstance();
-        $site = $sitesManager->getSiteFromId($idSite);
-
-        try {
-            return SitesManager::getTrackerExcludedQueryParameters($site);
-        } catch (Exception $e) {
-            // an exception is thrown when the user has no view access.
-            // do not throw the exception to the outside.
-            return array();
-        }
+        return Request::processRequest('SitesManager.getExcludedQueryParameters');
```

Comment:
Do we need to pass the parameter to maintain compatibility until the method is removed if someone somewhere still uses this (now deprecated) method?
```suggestion
        return Request::processRequest('SitesManager.getExcludedQueryParameters', ['idSite => $idSite']);
```
or
```suggestion
        return Request::processRequest('SitesManager.getExcludedQueryParameters', $idSite);
```

Reply:
No. `Request::processRequest` by default uses all request parameters from the original request. So as we want to perform another API action with the exact same parameter set, there is no need to overwrite any parameter.

Reply:
Thanks for the explanation :)